vehicle/src/test/scala/com/wedriveu/vehicle/control/VehicleBehavioursTest.scala

```diff
@@ -57,7 +57,7 @@ class VehicleBehavioursTest extends FunSuite with BeforeAndAfterEach {
   test("The vehicle position, after a random destination position input, should be equals to it") {
     randomLatitudeDestination = ThreadLocalRandom.current().nextDouble(minorBoundPositionLat, maxBoundPositionLat)
     randomLongitudeDestination = ThreadLocalRandom.current().nextDouble(minorBoundPositionLon, maxBoundPositionLon)
-    var vehicleBehaviours: VehicleBehaviours = new VehicleBehavioursImpl(vehicleControl.getVehicle())
+    var vehicleBehaviours: VehicleBehaviours = new VehicleBehavioursImpl(vehicleControl.getVehicle(), null)
```

Comment:
So, this `null` setting to `VehicleBehavioursImpl` constructor, what is for? Can it cause problems?